doc: concurrency of gc delete-range (#18359)
pingcap/docs: system-variables.md

- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Integer
- Default value: `-1`

> - Range: `{-1} ∪ [1, 256]`

Reviewer: ```suggestion
- Range: `-1` or `[1, 256]`
```

- Unit: Threads

> - This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the GC (Garbage Collection) process.

Reviewer: ```suggestion
- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
```

> - This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) and [Delete Range](/garbage-collection-overview.md#Delete-Ranges) steps of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.

Reviewer 2: ```suggestion
- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#Delete-Ranges) step of the GC process.
```

- By default, this variable is `-1`, allowing TiDB to automatically determine the appropriate number of threads based on workloads.
- When this variable is set to a number in the range of `[1, 256]`:
    - Resolve Locks directly uses the value set for this variable as the number of threads.

>     - Delete Range uses 1/4 of the value set for this variable as the number of threads.

Reviewer: ```suggestion
    - Delete Range uses one-fourth of the value set for this variable as the number of threads.
```
https://developers.google.com/style/slashes#slashes-with-fractions

- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Integer
- Default value: `-1`
- Range: `-1` or `[1, 256]`
- Unit: Threads
- This variable controls the number of concurrent threads during the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of the [Garbage Collection (GC)](/garbage-collection-overview.md) process.

> - Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#Delete-Ranges) step of the GC process.

Reviewer 2: ```suggestion
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#delete-ranges) step of the GC process.
```

> - Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Range](/garbage-collection-overview.md#delete-ranges) step of the GC process.

Reviewer 2: ```suggestion
- Starting from v8.3.0, this variable also controls the number of concurrent threads during the [Delete Ranges](/garbage-collection-overview.md#delete-ranges) step of the GC process.
```
